workers-site/index.js

```diff
@@ -1,5 +1,5 @@
 import { getAssetFromKV, mapRequestToAsset } from '@cloudflare/kv-asset-handler'
```

Comment:
What version of the @cloudflare/kv-asset-handler are you using? I don't see it in the package.json